interpreter/syntax/ast.ml

```diff
@@ -185,13 +185,9 @@ and instr' =
   | Unary of unop                     (* unary numeric operator *)
   | Binary of binop                   (* binary numeric operator *)
   | Convert of cvtop                  (* conversion *)
-  | TryCatch of block_type * instr list * (* try *)
-                (var * instr list) list * (* catch exception with tag *)
-                instr list option     (* catch_all *)
-  | TryDelegate of block_type * instr list * (* try *)
-                   var                (* delegate to outer handler *)
+  | Try of block_type * catch list * instr list  (* try *)
```

Comment:
Do you want to keep the name `TryTable` internally to better match the text format?

Reply:
Updated.